sdk/storage/azure-storage-blob/azure/storage/blob/aio/blob_client_async.py

```diff
@@ -292,6 +292,10 @@ def __init__(
             Use of customer-provided keys must be done over HTTPS.
             As the encryption key itself is provided in the request,
             a secure connection must be established to transfer the key.
+        :param int max_concurrency:
```

Comment:
same here...all keyword directives will be keyword instead of param.

I've updated in places other than this PR here (https://github.com/Azure/azure-sdk-for-python/pull/7855)